tools/configure/configure.ml

```diff
@@ -544,68 +99,26 @@ let camlbin, caml_version, camllib, findlib_version =
     let camllib, _ = run camlexec.find ["printconf";"stdlib"] in
     let camlbin = (* TODO beurk beurk beurk *)
```

Comment:
What does this comment mean?

Reply:
From 333d41a9a28767ab146aab0527ff2b235bbd31a7 , good luck :)

Reply:
We can probably remove this then?

Reply:
FWIW, it appears that "beurk" is French for "yuck".

Reply:
Ah OK, here I was thinking it was chicken noise.

Reply:
If Coq makes any sound at all, it ought to be a rooster crowing.  :-)